pkg/logs/input/docker/reader.go

```diff
@@ -10,16 +10,48 @@ package docker
 import (
 	"errors"
 	"io"
+	"time"
 )
 
 var errReaderNotInitialized = errors.New("reader not initialized")
 
+const defaultBackoffDuration = time.Second
```

Comment:
As discussed yesterday, I think we already have a backoff strategy in the tailer and I would rather reuse that one than adding a new one in the reader which we give more and more responsibilities.